### ISSUE TITLE
Viewport appears tiny when loading newly created level

### ISSUE THREAD

**Issue by greerdv** [o3de/o3de#7191]
**Describe the bug**
Viewport appears tiny when loading newly created level.

**Steps to reproduce**
1. Load newly created level
2. Note that viewport appears tiny (resizing the viewport will fix it)

**Expected behavior**
Viewport appears normal size.

**Actual behavior**
Viewport is tiny until resized.

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/82226198/151361047-20fd030c-ffe8-4f90-90a6-7fd665b490e8.png)

**Found in Branch**
development 7a45aa5bbf

**Comment by spham-amzn** [comment]
This is an issue with Vulkan (on Linux and Windows)

**Comment by greerdv** [comment]
@jonawals Yeah, I was just able to reproduce this by making a new level. I'm pretty much up to date with latest development. I have --RHI=Vulkan in my command arguments when running the editor.
